CHANGELOG.rst

```diff
@@ -1,5 +1,51 @@
 .. _changelog:
 
+0.23.0
+------
+
+Renku ``0.23.0`` introduces the Renku Workflow File, a friendlier way to encode and run your data analysis pipelines
+on Renku. This yaml-based file is easy to write and execution is as easy as `renku run workflow.yml`. Add a workflow
+to your Renku project with our docs [here](https://renku.readthedocs.io/en/stable/topic-guides/workflows/workflow-file.html)!
+
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+ * 📝 **Renku CLI**: Add support for writing workflow files which can be executed directly, greatly improving the UX
```

Comment:
Does the CLI add support for writing workflow files or just parsing and executing them?

Reply:
both

Reply:
I would leave the word "writing" out, if I understood @seanrmurphy 's comment correctly

Reply:
I just cannot imagine what is meant here - write a file based on what? Does it pull it down from somewhere? where does it write it to? Is it some kind of user based interaction to generate the file?

Also, it's not completely clear to me whether the execution is part of the change (I guess not).

If writing a file is the essence of the change and execution already existed, I guess we can't really leave out writing.

Is this somehow what is meant?

`Add support for CLI-based interactive workflow file generation`

WDYT @Panaetius ?

Reply:
@seanrmurphy Ralf already edited the line, it now reads `*Add support for workflow files which can be executed directly, greatly improving the UX around workflows`